setup.py

```diff
@@ -33,13 +45,7 @@
     namespace_packages=['collective'],
     include_package_data=True,
     zip_safe=False,
-    install_requires=[
-        'setuptools',
-        'simplejson',
-        'collective.transmogrifier',
```

Comment:
I see `collective.transmogrifier` has disappeared. Was this on purpose, because we are maybe not actually using it?

Otherwise: the change looks good to me.